<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cooper</family>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(52, 143, 128);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="HeapSortButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string>Método HeapSort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuickSortButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string>Método QuickSort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShellSortButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>100</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string>Método ShellSort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Introduce los datos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 130, 130);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apellido :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Nota :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellido">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 130, 130);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNota">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 217, 217);background-color: rgb(130, 130, 130);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>271</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>271</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>271</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>271</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string>Añadir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Métodos de ordenamiento</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
